packages/beacon-node/src/chain/archiveStore/constants.ts

```diff
@@ -0,0 +1,5 @@
+import {ArchiveMode} from "./interface.js";
+
+export const DEFAULT_ARCHIVE_MODE = ArchiveMode.Frequency;
+
+export const PROCESS_FINALIZED_CHECKPOINT_QUEUE_LEN = 256;
```

Comment:
this variable is already long, might as well not abbreviate length
```suggestion
export const PROCESS_FINALIZED_CHECKPOINT_QUEUE_LENGTH = 256;
```